src/systems/tracked-controls.js

```diff
@@ -7,6 +7,7 @@ var utils = require('../utils');
  * It maintains a list with the available tracked controllers
  */
 module.exports.System = registerSystem('tracked-controls', {
+  schema: {alwaysAllowIdPrefixes: {type: 'array'}},
```

Comment:
There is a spec https://w3c.github.io/gamepad/extensions.html#gamepadpose-interface